product_expiry_simple/models/stock_production_lot.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# © 2017 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api
+
+
+class StockProductionLot(models.Model):
+    _inherit = 'stock.production.lot'
+    _rec_name = 'display_name'
+
+    expiry_date = fields.Date(string='Expiry Date')
+    display_name = fields.Char(
+        compute='compute_display_name_field',
```

Comment:
Just a nitpicking request: change compute method name respecting [contrib rule](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#field) for such field: `Compute Field: the compute method pattern is _compute_<field_name>`